chart/templates/manager-rbac.yaml

```diff
@@ -212,6 +212,38 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+    - secrets.hashicorp.com
+  resources:
+    - hcpauths
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - secrets.hashicorp.com
+  resources:
+    - hcpauths/status
+  verbs:
+    - get
+- apiGroups:
+    - secrets.hashicorp.com
+  resources:
+    - hcpauths
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - secrets.hashicorp.com
+  resources:
+    - hcpauths/status
+  verbs:
+    - get
```

Comment:
I forget if this is autogenerated, but it looks like these permissions (lines 233-246) are already included above.

Reply:
Interesting. They are not autogenerated, unfortunately.